src/blog/2024/01/sentiment-analysis-with-node-red.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Sentiment Analysis with Node-RED
+subtitle: A guide to building a simple sentiment analysis system with Node-RED.
+description: A guide to building a simple sentiment analysis system with Node-RED.
+date: 2024-01-23
+authors: ["sumit-shinde"]
+image: /blog/2024/01/images/sentiment-analysis-dashboard-gif.gif
+tags:
+    - posts
+    - node-red
+    - dashboard
+---
+
+Have you ever built a sentiment analysis system to extract insights from text content? If yes then I don’t think you'll need an explanation of how complex it is to build. If you haven't built then be with us. Today in this guide, we will build a sentiment analysis system with Node-RED.
+
+<!--more-->
+
+# What exactly is sentiment analysis?
+
+Sentiment analysis is a context-mining technique used to understand emotions and opinions expressed in text, often classifying them as positive, neutral, or negative. There are many real-world applications of this technique.
+
+- **Analysing Feedback:** Customers, or other stakeholders like employees, are periodically requested to fill out a feedback form. Analysis of such feedback is the most widespread application of sentiment analysis.
+- **Campaign Monitoring:** Another use-case of sentiment analysis is a measure of influence which is crucial in any marketing campaign.
+- **Brand Monitoring:** Brand monitoring is another great use case for sentiment analysis. Companies can use sentiment analysis to check the social media sentiments around their brand from their audience.
+
+# Building a Form in Dashboard 2.0
+
+In this system, we will analyse the sentiment of text content obtained from the user, for that we are going to build a user interface using Dashboard 2.0 and Node-RED.
+
+1. Install Node-RED Dashboard 2.0. Follow these [instructions](https://dashboard.flowfuse.com/getting-started.html) to install.
+2. Create your first group, page, theme, and base.
+3. Drag a ui form widget to the canvas and select the created group.
```

Comment:
```suggestion
3. Drag an ui form widget to the canvas and select the created group.
```

Reply:
@Hasmin-AC I think A is better: https://www.quora.com/Which-is-the-correct-usage-a-user-or-an-user#:~:text=It's%20'A%20user'.,sound%20and%20not%20the%20alphabet.

Reply:
Sure, we can follow that convention.